src/firefly/js/ui/WorkspaceViewer.jsx

```diff
@@ -458,40 +458,37 @@ function showWorkspaceAsPopup({onComplete, value, fieldKey=workspaceUploadDef.fi
         const showMask = isAccessWorkspace();
         const popup = (
             <PopupPanel title={'Read file from workspace'}>
+
                 <div style={popupPanelResizableStyle}>
-                    <FieldGroup style={{height: 'calc(100% - 80px)', width: '100%', position: 'relative'}}
-                                groupKey={workspacePopupGroup} keepState={true}>
-                        <div style={style}>
-                            <WorkspaceViewField fieldKey={fieldKey}
+                    <FieldGroup groupKey={workspacePopupGroup} keepState={true}>
+                        <Stack spacing={2}
+                               sx={{ps:2, my:2, justifyContent: 'space-between'}}>
```

Comment:
`ps`? Also, you should check to see if you really need to use margin for Y but, padding for X.